openpype/modules/deadline/plugins/publish/submit_nuke_deadline.py

```diff
@@ -32,7 +32,7 @@ class NukeSubmitDeadline(pyblish.api.InstancePlugin,
     label = "Submit Nuke to Deadline"
     order = pyblish.api.IntegratorOrder + 0.1
     hosts = ["nuke"]
-    families = ["render", "prerender"]
```

Comment:
@jakubjezek001 that's true, in fact I have changed it on our fork of OP since but because there was no track on this PR I forgot to update it back here as well sorry. However, in order to avoid this plugin running for `farm_frames` what I did is add this on the `def process` function:
```python
        # If render target is set to farm_frames we skip the
        # plugin
        render_target = instance.data.get("render_target")
        if render_target == "farm_frames":
            return
```
I can push those changes if you want

Reply:
Although I'm not sure why I had to add this, in theory the existing
```python
        if not instance.data.get("farm"):
            self.log.debug("Skipping local instance.")
            return
```
should already address that... so let's test without it

Reply:
done